test/index.js

```diff
@@ -247,6 +247,8 @@ describe('Metalsmith', function(){
     });
 
     it('should traverse a symbolic link to a directory', function(done){
+      // symbolic links are *nix specific
+      if (process.platform === 'win32') { return done(); }
```

Comment:
FWIW, symbolic links actually _do_ exist in NTFS, and you can create them on Windows with the [`mklink` command](https://technet.microsoft.com/en-us/library/cc753194(v=ws.11).aspx) if you have Adminstrative privileges. Not sure if we want to support that though. In any case, it's definitely not something that needs to be done in this PR.